src/test/java/org/craftercms/studio/impl/v2/repository/blob/BlobAwareContentRepositoryTest.java

```diff
@@ -25,7 +25,7 @@
 import org.craftercms.studio.api.v1.to.VersionTO;
 import org.craftercms.studio.api.v2.repository.blob.StudioBlobStore;
 import org.craftercms.studio.api.v2.repository.blob.StudioBlobStoreResolver;
-import org.craftercms.studio.impl.v2.repository.GitContentRepository;
+import org.craftercms.studio.impl.v1.repository.git.GitContentRepository;
```

Comment:
Why?

Reply:
This was caused by a bad merge and the test was not compiling